main.go

```diff
@@ -51,7 +52,7 @@ func runtask(filename string) {
 		Command: []*string{
 			aws.String("--url"),
 			esurl,
-			aws.String("ingest"),
+			aws.String("ingest --debug --auto"),
```

Comment:
Though I haven't tried it like what you did, I'm almost certain these need to be separate, like:
```
aws.String("ingest"),
aws.String("--debug"),
aws.String("--auto"),
```

Reply:
will fix